modules/tensor_mechanics/src/materials/ComputeMultipleInelasticStress.C

```diff
@@ -167,12 +172,44 @@ ComputeMultipleInelasticStress::initialSetup()
                  "operator are being combined. Either set tangent_operator to elastic, implement "
                  "the corrent tangent formulations, or use different models.");
   }
+
+  if (_include_damage)
+  {
+    MaterialName damage_model_name = getParam<MaterialName>("damage_model");
+    DamageBase * dmb = dynamic_cast<DamageBase *>(&getMaterialByName(damage_model_name));
+    if (dmb)
+      _damage_model = dmb;
+    else
+      paramError("damage_model",
+                 "Damage Model " + damage_model_name +
+                     " is not compatible with ComputeMultipleInelasticStress");
+  }
 }
 
 void
 ComputeMultipleInelasticStress::computeQpStress()
 {
+  if (_include_damage)
```

Comment:
```suggestion
  if (_ damage_model)
```

here and everywhere below